### PR TITLE
Change Mocker not to call anvil_mine

### PR DESCRIPTION
**Pull request by antonyshchenko**
Calling `anivl_mine` breaks data integrity in Anvil when running in envs with auto-mining enabled (demo)

### CODE REVIEW

File: integrationtests/src/main/kotlin/co/chainring/integrationtests/utils/Wallet.kt [hunk 1]
```diff
@@ -71,13 +74,25 @@ class Wallet(
         return loadErc20Contract(symbol).balanceOf(address.value).send()
     }
 
-    fun mintERC20(symbol: String, amount: BigInteger) {
-        loadErc20Contract(symbol).mint(address.value, amount).sendAndWaitForConfirmation()
+    fun mintERC20AndMine(symbol: String, amount: BigInteger): TransactionReceipt {
```
Comment (imykolenko) [inline integrationtests/src/main/kotlin/co/chainring/integrationtests/utils/Wallet.kt:77]: nice renaming 👍

File: integrationtests/src/main/kotlin/co/chainring/integrationtests/utils/TestBlockchainClient.kt [hunk 2]
```diff
@@ -3,28 +3,47 @@ package co.chainring.integrationtests.utils
...
-        ).sendAndWaitForConfirmation()
+//    fun depositNative(address: Address, amount: BigInteger): TransactionReceipt {
+//        return Transfer(web3j, transactionManager).sendFunds(
+//            address.value,
+//            Convert.toWei(amount.toString(10), Convert.Unit.WEI),
+//            Convert.Unit.WEI,
+//            web3j.ethGasPrice().send().gasPrice,
+//            gasProvider.gasLimit,
+//        ).send()
+//    }
```
Comment (imykolenko) [inline integrationtests/src/main/kotlin/co/chainring/integrationtests/utils/TestBlockchainClient.kt:30]: is this code still needed? If no - let's drop it

**Review by imykolenko — APPROVED** [review]
looks great

File: integrationtests/src/main/kotlin/co/chainring/integrationtests/utils/TestBlockchainClient.kt (hunk 2 above)
Comment (antonyshchenko) [reply integrationtests/src/main/kotlin/co/chainring/integrationtests/utils/TestBlockchainClient.kt]: yup, thanks!

**Pull request merged**